jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/ButtonToggleTest.java

```diff
@@ -0,0 +1,84 @@
+
+        buttonToggleSection.basicButtonToggleUnderline.isDisplayed();
+        buttonToggleSection.basicButtonToggleUnderline.click();
+        buttonToggleSection.basicButtonToggleUnderline.has().attr("aria-pressed", "true");
+        buttonToggleSection.basicButtonToggleUnderline.click();
+        buttonToggleSection.basicButtonToggleUnderline.has().attr("aria-pressed", "false");
+    }
+
+    @Test
+    public void basicButtonToggleNamesTest() {
+        buttonToggleSection.basicButtonToggleBold.has().text("Bold");
+        buttonToggleSection.basicButtonToggleItalic.has().text("Italic");
+        buttonToggleSection.basicButtonToggleUnderline.has().text("Underline");
+    }
+
+    @Test
+    public void pressAllBasicButtonTogglesTest() {
+        buttonToggleSection.basicButtonToggleBold.isDisplayed();
+        buttonToggleSection.basicButtonToggleBold.click();
+        buttonToggleSection.basicButtonToggleItalic.isDisplayed();
+        buttonToggleSection.basicButtonToggleItalic.click();
+        buttonToggleSection.basicButtonToggleUnderline.isDisplayed();
+        buttonToggleSection.basicButtonToggleUnderline.click();
+        buttonToggleSection.basicButtonToggleBold.has().attr("aria-pressed", "true");
+        buttonToggleSection.basicButtonToggleItalic.has().attr("aria-pressed", "true");
+        buttonToggleSection.basicButtonToggleUnderline.has().attr("aria-pressed", "true");
+    }
+
+    @Test
+    public void exclusiveButtonToggleTest() {
+        buttonToggleSection.exclusiveButtonToggleLeft.isDisplayed();
+        buttonToggleSection.exclusiveButtonToggleLeft.click();
+        buttonToggleSection.exclusiveButtonToggleLeft.has().attr("aria-pressed", "true");
+        buttonToggleSection.selectedValue.has().text("Selected value: left");
+
+        buttonToggleSection.exclusiveButtonToggleCenter.isDisplayed();
+        buttonToggleSection.exclusiveButtonToggleCenter.click();
+        buttonToggleSection.exclusiveButtonToggleCenter.has().attr("aria-pressed", "true");
+        buttonToggleSection.exclusiveButtonToggleLeft.has().attr("aria-pressed", "false");
+        buttonToggleSection.selectedValue.has().text("Selected value: center");
+
+        buttonToggleSection.exclusiveButtonToggleRight.isDisplayed();
+        buttonToggleSection.exclusiveButtonToggleRight.click();
+        buttonToggleSection.exclusiveButtonToggleRight.has().attr("aria-pressed", "true");
+        buttonToggleSection.exclusiveButtonToggleCenter.has().attr("aria-pressed", "false");
+        buttonToggleSection.selectedValue.has().text("Selected value: right");
+
+        buttonToggleSection.exclusiveButtonToggleJustify.isDisplayed();
+        buttonToggleSection.exclusiveButtonToggleJustify.is().disabled();
+
+        buttonToggleSection.exclusiveButtonToggleLeft.click();
+        buttonToggleSection.exclusiveButtonToggleRight.has().attr("aria-pressed", "false");
+    }
+}
```

Comment:
Please try to use method .is().enabled()

Reply:
I don't understand, where should I use this method?